assets/src/edit-story/elements/text/edit.js

```diff
@@ -45,28 +45,14 @@ import useUnmount from '../../utils/useUnmount';
 import createSolid from '../../utils/createSolid';
 import stripHTML from '../../utils/stripHTML';
 import calcRotatedResizeOffset from '../../utils/calcRotatedResizeOffset';
+import { useTransformHandler } from '../../components/transform';
 import { generateParagraphTextStyle, getHighlightLineheight } from './util';
 
 // Wrapper bounds the text editor within the element bounds. The resize
 // logic updates the height of this element to show the new height based
 // on the content and properties.
-// Background color is used to make the edited element more prominent and
-// easier to see.
 const Wrapper = styled.div`
   ${elementFillContent}
-  background-color: ${({ theme }) => theme.colors.whiteout};
```

Comment:
This, imho, belongs in a separate pull request.

Reply:
Leaving this in would require ensuring the Wrapper is also moving together with the Moveable (currently it's not) and would need extra handling for ensuring that.

Since we needed to remove this anyway, it seemed to make sense to remove this here as opposed to adding additional support for it here to just remove this again in #2040 

Thoughts?